src/app/helptext/system/ssh-connections.ts

```diff
@@ -0,0 +1,38 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T(''),
+
+    setup_method_placeholder: T('Setup Method'),
+    setup_method_tooltip: T(''),
+
+    host_placeholder: T('Host'),
+    host_tooltip: T(''),
+
+    port_placeholder: T('Port'),
+    port_tooltip: T(''),
+
+    username_placeholder: T('Username'),
+    username_tooltip: T(''),
+
+    private_key_placeholder: T('Private Key'),
+    private_key_tooltip: T(''),
```

Comment:
Connection key used to authenticate with the remote system. Choose a private key generated from a local Certificate Authority.

Reply:
Private key is local, the keys are generated locally and the public key is what goes to the remote system.